README.md

```diff
@@ -45,55 +45,69 @@ The following are also relevant for consistency, but not critical:
 
 ## Backup
 
-To backup, prepare to receive the backup data in the terminal:
-
-```sh
-adb reverse tcp:12101 tcp:12101
-nc -l -s 127.0.0.1 -p 12101 > firefox-android-backup.tar.gz
-```
-
-Open the Firefox app on your phone, and visit `about:blank` (any webpage works).
-Copy the contents of [`snippets_for_firefox_debugging.js`](snippets_for_firefox_debugging.js).
-Open `about:debugging`, scroll down to "Main Process" and click on Inspect.
-Switch to the Console, paste the code and run it. Then type and run:
-
-```js
-fab_backup_create();
-```
-
-This copies [relevant files](#relevant-files) to `firefox-android-backup.tar.gz`
-without changing any data, except for one log file. To remove the log file, run:
-
-```js
-fab_cleanup();
-```
+Optionally before performing the backup clear the app cache of the Firefox app
+to reduce the size of the backup.
+
+1. To backup, prepare to receive the backup data in the terminal:
+    ```sh
+    adb reverse tcp:12101 tcp:12101
+    nc -l -s 127.0.0.1 -p 12101 > firefox-android-backup.tar.gz
+    ```
+
+    On MacOS you might have to use `nc -l 127.0.0.1 12101 > ...` ([issue #3](https://github.com/Rob--W/firefox-android-backup-restore/issues/3)).\
+    On Windows you can use [ncat](https://nmap.org/ncat/) with `ncat -l 127.0.0.1 -p 12101 > ...`.
+
+3. Open the Firefox app on your phone, and visit `about:blank` (any webpage works).
+4. On your PC, open `about:debugging` in Firefox
+5. Set up a debug connection to the Android app, as described in the [requirements section](#requirements)
+6. On your PC, for the debugged Android app, scroll down to "Main Process", click on "Inspect" and open the "Console" tab.
+7. Copy the contents of [`snippets_for_firefox_debugging.js`](snippets_for_firefox_debugging.js).
+8. Paste the code you copied into the Console run it.
+9. In the Console run the following to make sure the script is functional:
+    ```js
+    fab_sanity_check();
+    ```
+10. Then type and run:
+    ```js
+    fab_backup_create();
+    ```
+
+    This copies [relevant files](#relevant-files) to `firefox-android-backup.tar.gz`.
+    Wait until "Backup successfully created and transferred!" appears on the Console.
+
+    If the script fails, saying it cannot find `nc`, see [issue #2](https://github.com/Rob--W/firefox-android-backup-restore/issues/2).
+11. Close the reverse connection you opened earlier for the transfer:
+     ```sh
+     adb reverse –-remove tcp:12101
```

Comment:
with my linux adb version 35.0.2-android-tools:
```
$ adb reverse –-remove tcp:12101
adb: error: cannot bind listener: unknown socket specification:–-remove
```
but it works with
```
$ adb reverse tcp:12101 --remove
```

Reply:
Thanks for mentioning this! It looks like I accidentally used an "en dash" (`–`, U+2013) instead of a regular hyphen here in the documentation. Maybe that was the issue.

Could you please check again?

Reply:
When debugging, I re-typed in reverse order, accidentally fixing the copy-pasted en-dash. I wrongly assumed the issue was the args order.
I can't test presently, but your fix of en-dash does make totally sense.